tensorflow/core/kernels/mkl/mkl_qmatmul_op_test.cc

```diff
@@ -12,7 +12,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-#if defined(INTEL_MKL) && !defined(ENABLE_ONEDNN_V3)
+#if defined(INTEL_MKL)
```

Comment:
Nit: Please also remove `&& !ENABLE_ONEDNN_V3` in the comment at the end of this file.
```
#endif  // INTEL_MKL && !ENABLE_ONEDNN_V3
```

Reply:
Fixed.